test-partner/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
 RUN dnf install -y wget make hostname iproute iputils openssh openssh-clients podman psmisc
-
+RUN dnf remove -y iptables ip6tables nftables 
```

Comment:
podman depends on and installs iptables ip6tables nftables. Do we use podman in the image? If not remove podman from the installation list, firewall utilities will not be installed.

Reply:
I believe we need `podman` installed for our `fsdiff` test in the `cnf-certification-test` repo.

Reply:
ok thanks for pointing this. I will remove the PR